examples/big-bang/zarf.yaml

```
@@ -27,3 +27,10 @@ components:
           - config/loki.yaml
           # Values are merged in order, so this would override the above and disable everything if uncommented
           # - config/disable-all.yaml
+  - name: extra-virtual-services
+    required: true
```

Can we name this more specifically and make it optional for those who want to keep Gitea internal or are using an external git-repo?

```suggestion
  - name: gitea-virtual-service
    description: >
      Expose the internal Zarf Gitea server through the Big Bang Istio deployment via a virtual service.
      (only applies if you are using the Zarf-provided Gitea deployment - not an externally configured git host)
```